[2025-05-02 00:04:48,618 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 00:04:48,619 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 00:10:11,222 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 00:10:11,223 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 00:16:05,342 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 00:16:05,344 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 00:21:43,048 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 00:21:43,049 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 00:27:02,360 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 00:27:02,362 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 00:32:41,231 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 00:32:41,233 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 00:38:30,159 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 00:38:30,161 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 00:44:17,004 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 00:44:17,005 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 00:49:52,802 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 00:49:52,804 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 00:55:21,300 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 00:55:21,301 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:00:12,357 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:00:12,359 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:04:55,610 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:04:55,612 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:09:50,287 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:09:50,289 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:14:28,440 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:14:28,441 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:18:47,355 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:18:47,356 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:23:46,983 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:23:46,985 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:28:28,318 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:28:28,320 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:33:35,249 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:33:35,251 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:38:45,257 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:38:45,259 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:43:42,461 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:43:42,462 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:48:36,598 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:48:36,600 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 01:53:12,965 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 01:53:12,967 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 01:57:31,386 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 01:57:31,387 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:01:53,338 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:01:53,339 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:06:29,094 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:06:29,095 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:11:34,934 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:11:34,936 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:16:10,008 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:16:10,010 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:20:58,856 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 02:20:58,857 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 02:25:38,605 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 02:25:38,607 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 02:30:16,660 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:30:16,662 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:46:29,018 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 02:46:29,020 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 02:51:55,903 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 02:51:55,904 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 02:57:06,785 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 02:57:06,786 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 03:01:47,673 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:01:47,675 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:07:16,465 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 03:07:16,467 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 03:12:30,495 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 03:12:30,497 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 03:17:40,624 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:17:40,625 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:22:59,030 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:22:59,031 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:27:29,297 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 03:27:29,299 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 03:32:17,215 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:32:17,217 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:37:06,001 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 03:37:06,003 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 03:42:05,307 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:42:05,309 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:46:38,073 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:46:38,074 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:51:12,159 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:51:12,161 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 03:55:41,135 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 03:55:41,137 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 04:00:45,373 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 04:00:45,375 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 04:05:15,723 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:05:15,724 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:10:05,321 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:10:05,323 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:14:54,392 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 04:14:54,393 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 04:20:07,230 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 04:20:07,232 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 04:25:14,979 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:25:14,981 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:30:34,962 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 04:30:34,964 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 04:35:52,935 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:35:52,937 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:41:16,419 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:41:16,421 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:46:37,675 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:46:37,676 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:51:39,918 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 04:51:39,920 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 04:56:00,345 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 04:56:00,346 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:01:06,383 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:01:06,384 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:05:45,292 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:05:45,294 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:10:37,527 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:10:37,528 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:15:30,355 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:15:30,357 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 05:20:14,309 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:20:14,310 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 05:24:46,046 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:24:46,048 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 05:29:45,153 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:29:45,155 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:34:57,723 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:34:57,724 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:39:51,351 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:39:51,353 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:44:33,079 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:44:33,080 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 05:48:46,669 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:48:46,670 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 05:53:34,096 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 05:53:34,098 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 05:58:43,613 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 05:58:43,615 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 06:03:11,379 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 06:03:11,380 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 06:08:29,596 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:08:29,598 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:13:55,795 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:13:55,797 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:18:48,681 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:18:48,682 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:23:30,846 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:23:30,848 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:28:10,947 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:28:10,949 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:32:53,572 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:32:53,574 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:37:28,711 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:37:28,713 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:41:57,296 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:41:57,297 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:46:54,983 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 06:46:54,985 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 06:51:39,861 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 06:51:39,862 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 06:56:32,694 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 06:56:32,696 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:00:58,242 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 07:00:58,244 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 07:05:58,885 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 07:05:58,886 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 07:10:59,814 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 07:10:59,815 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 07:15:44,019 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:15:44,020 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:20:24,309 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:20:24,310 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:24:42,279 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:24:42,281 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:29:20,541 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:29:20,543 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:34:13,189 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:34:13,190 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:39:00,458 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:39:00,460 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:44:04,019 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:44:04,021 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:49:12,642 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 07:49:12,643 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 07:54:20,649 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 07:54:20,651 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 07:58:57,665 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 07:58:57,666 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:03:28,025 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 08:03:28,027 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 08:08:05,140 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 08:08:05,142 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 08:13:16,696 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:13:16,697 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:17:55,370 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:17:55,371 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:22:57,290 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 08:22:57,292 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 08:27:45,674 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 08:27:45,675 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 08:32:20,343 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:32:20,346 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:36:56,088 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:36:56,089 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:41:38,743 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:41:38,745 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:46:28,242 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:46:28,244 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:51:45,113 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:51:45,115 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 08:56:17,815 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 08:56:17,817 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 09:00:58,490 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 09:00:58,492 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-02 09:05:31,449 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-02 09:05:31,451 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-02 09:10:19,510 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-02 09:10:19,512 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
